rs_lib/src/common.rs

```diff
@@ -5,6 +5,9 @@ use std::collections::HashMap;
 
 use url::Url;
 
+// TODO(ry) HTTP headers are not unique key, value pairs. There may be more than
+// one header line with the same key. This should be changed to something like
+// Vec<(String, String)>
```

Comment:
Moved this todo from the CLI.

Reply:
🤷 it's been 5 years or more since it was added. I don't think we've run into the problems with it, so it might not be worth pulling it.